### ISSUE TITLE
[bactopia-tool] ERROR ~ Unknown method invocation `startsWith` on Boolean type

### ISSUE THREAD

**Issue by yzhzu** [bactopia/bactopia#445]
Core Nextflow options
  runName          : curious_booth
  container        : quay.io/bactopia/bactopia:3.0.0
  launchDir        : /home/shsmu/Data/Temp/Results
  workDir          : /home/shsmu/Data/Temp/Results/work
  projectDir       : /home/shsmu/anaconda3/envs/bactopia/share/bactopia-3.0.0
  userName         : shsmu
  profile          : standard
  configFiles      : /home/shsmu/anaconda3/envs/bactopia/share/bactopia-3.0.0/nextflow.config

Required Parameters
  bactopia         : true

Filtering Parameters
  include          : AT19977.fasta

Optional Parameters
  outdir           : true
  run_name         : bakta

Nextflow Profile Parameters
  condadir         : /home/shsmu/.bactopia/conda
  datasets_cache   : /home/shsmu/.bactopia/datasets
  singularity_cache: /home/shsmu/.bactopia/singularity

Helpful Parameters
  wf               : bakta

!! Only displaying parameters that differ from the pipeline defaults !!
--------------------------------------------------------------------
If you use bactopia for your analysis please cite:

* Bactopia
  https://doi.org/10.1128/mSystems.00190-20

* The nf-core framework
  https://doi.org/10.1038/s41587-020-0439-x

* Software dependencies
  https://bactopia.github.io/acknowledgements/
--------------------------------------------------------------------
WARN: 
WARN: Conda Disclaimer
WARN: 
WARN: If you have access to Docker or Singularity, please consider
WARN: running Bactopia using containers. The containers are less
WARN: susceptible to Conda environment related issues (e.g. version
WARN: conflicts) and errors caused by creation of conda environments
WARN: in parallel (use '--max_cpus 1' to over come this error).
WARN: 
WARN: To use containers, you can use the profile parameter
WARN:     Docker: -profile docker
WARN:     Singularity: -profile singularity
WARN: 
--------------------------------------------------------------------
ERROR ~ Unknown method invocation `startsWith` on Boolean type

 -- Check script '/home/shsmu/anaconda3/envs/bactopia/share/bactopia-3.0.0/./workflows/bactopia-tools.nf' at line: 20 or see '.nextflow.log' file for more details

**Comment by rpetit3** [comment]
Hi @yzhzu 

I've released v3.0.1 which might have fixed this. For now I will close this, please feel free to reopen.

Cheers,
Robert

**Comment by kimleeng** [comment]
I'm getting this with 3.20 with run command
`bactopia -profile docker run --sample AMA170 --SE ./AMA170.q12.Mbp500.fastq.gz --ont`

```
ERROR ~ Unknown method invocation `startsWith` on Boolean type

 -- Check script '/Users/b246297/playground/bactopia/.pixi/envs/default/share/bactopia-3.2.0/./workflows/bactopia.nf' at line: 19 or see '.nextflow.log' file for more details
```

and .nextflow.log content of:
```
May-07 10:35:21.209 [main] DEBUG nextflow.script.ScriptRunner - Parsed script files:
  Script_2f47e363e7b356fd: /Users/b246297/playground/bactopia/.pixi/envs/default/share/bactopia-3.2.0/./workflows/bactopia.nf
  Script_28b956a88b5bebd6: /Users/b246297/playground/bactopia/.pixi/envs/default/share/bactopia-3.2.0/main.nf
  Script_e2a6881b4773ed6b: /Users/b246297/playground/bactopia/.pixi/envs/default/share/bactopia-3.2.0/./lib/nf/functions.nf
  Script_5006a0b829d2df9f: /Users/b246297/playground/bactopia/.pixi/envs/default/share/bactopia-3.2.0/./workflows/../lib/nf/bactopia.nf
May-07 10:35:21.212 [main] DEBUG nextflow.Session - Session aborted -- Cause: No signature of method: java.lang.Boolean.startsWith() is applicable for argument types: (String) values: [gs://]
May-07 10:35:21.218 [main] ERROR nextflow.cli.Launcher - @unknown
groovy.lang.MissingMethodException: No signature of method: java.lang.Boolean.startsWith() is applicable for argument types: (String) values: [gs://]
	at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.unwrap(ScriptBytecodeAdapter.java:70)
	at org.codehaus.groovy.runtime.callsite.PojoMetaClassSite.call(PojoMetaClassSite.java:46)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
	at Utils.isLocal(Utils.groovy:83)
	at Utils$isLocal.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
	at WorkflowBactopia.initialise(WorkflowBactopia.groovy:60)
	at WorkflowBactopia$initialise.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:166)
	at Script_2f47e363e7b356fd.runScript(Script_2f47e363e7b356fd:19)
	at nextflow.script.BaseScript.run0(BaseScript.groovy:144)
	at nextflow.script.BaseScript.run(BaseScript.groovy:192)
	at nextflow.script.ScriptParser.runScript(ScriptParser.groovy:236)
	at nextflow.script.ScriptParser.runScript(ScriptParser.groovy:222)
	at nextflow.script.IncludeDef.memoizedMethodPriv$loadModule0PathMapSession(IncludeDef.groovy:151)
	at nextflow.script.IncludeDef.access$0(IncludeDef.groovy)
	at nextflow.script.IncludeDef$__clinit__closure2.doCall(IncludeDef.groovy)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1030)
	at groovy.lang.Closure.call(Closure.java:427)
	at org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)
	at org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)
	at org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)
	at org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)
	at nextflow.script.IncludeDef.loadModule0(IncludeDef.groovy)
	at nextflow.script.IncludeDef.load0(IncludeDef.groovy:123)
	at nextflow.script.IncludeDef$load0$1.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
	at Script_28b956a88b5bebd6.runScript(Script_28b956a88b5bebd6:16)
	at nextflow.script.BaseScript.run0(BaseScript.groovy:144)
	at nextflow.script.BaseScript.run(BaseScript.groovy:192)
	at nextflow.script.ScriptParser.runScript(ScriptParser.groovy:236)
	at nextflow.script.ScriptRunner.run(ScriptRunner.groovy:242)
	at nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:137)
	at nextflow.cli.CmdRun.run(CmdRun.groovy:372)
	at nextflow.cli.Launcher.run(Launcher.groovy:500)
	at nextflow.cli.Launcher.main(Launcher.groovy:672)
```
